pkg/convert/go.go

```diff
@@ -19,7 +19,7 @@ func FromGoMessages(m model.Messages) ([]byte, error) {
 	return msg, nil
 }
 
-func ToGoMessages(b []byte) (model.Messages, error) {
+func FromGo(b []byte) (model.Messages, error) {
```

Comment:
Refactored so it's consistent with other converts

From: bytes→messages
To: messages→bytes